### ISSUE TITLE
LibreOffice application abscense or malfunction

### ISSUE THREAD

**Issue by DanteChambers** [QubesOS/qubes-issues#5995]
### Affected component(s) or functionality:

The libreOffice Suite.  6.4   AppVM made with Fedora or Debian 10

---

### Steps to reproduce the behavior:

Create new VM
Use template VM
Chose Fedora 32 or Debian 10
Use Snapcraft to install LibreOffice  or Use flatpak to LibreOffice
Close appVM
Restart AppVM by accessing the qubes setting application 
Refresh applications
Exit Qube settings 
access any LibreOffice application

### Expected or desired behavior:

I was expecting the desired libreOffice application to open. 

### Actual behavior:

The disired application did not open, inspite of waiting. 

### General notes:

Flatpak is the application called software in fedora. 
It can also be accessed from the flatpak site.

It would be nice, to have the libre office application working considering it should work on all plateforms as before however, this issue started about 3 to 5 days ago after a dom0 and or template update.  

I also tried to download the latest version from libreoffice.org and create an app Image but that failled as well.  

What happened to the application's functioning between this week and last week?  

---

### I have consulted the following relevant [documentation](https://www.qubes-os.org/doc/):
With the various forums such as Debian and unfortunately I cannot consult the libreOffice forum as they have a strong spam policy which prevents me from using Whonix.

**Comment by andrewdavidwong** [comment]
What happens if you type `sudo dnf install libreoffice` in a Fedora VM terminal? This is the standard way to install it.

**Comment by DanteChambers** [comment]
Sorry for the delay, I was hoping to have positive news for you.  
A few days ago, I tired your suggestion and unfortunately it did not work.  So over the weekend I tried a few other codes and added some common code a template. That did work either.   

Today, I was able to perform the suggested code and it downloaded.  However, it's not installing itself in the application menu of the qube settings-app.  I'll try restarting the appVM a few more times today and see.  

Thanks for the suggestion, at least it's downloaded.  The next step will be to have it work.
